resharper/resharper-unity/src/Rider/Protocol/BackendUnityProtocol.cs

```diff
@@ -216,7 +216,7 @@ private void OnOutOfSync(Lifetime lifetime)
             else
             {
                 var notification = new NotificationModel("Advanced Unity integration is unavailable",
-                    $"Please update External Editor to {myHostProductInfo.VersionMarketingString} in Unity Preferences.",
+                    $"Make sure Rider {myHostProductInfo.VersionMarketingString} is set as the External Editor in Unity.",
```

Comment:
"Make sure Rider {…} is set as the External Editor in Unity preferences."

Matches the notification below